### PR TITLE
Fix #66 update travis to test only python 3.7 and also run the docker…

### LINKED ISSUE #66: Remove python 2.7 from travis CI
CI build is failing for python 2.7 on travis.   Looking at updates to CHANGES.md for release  v3.1.2 is looks like python2.7 is no longer supported, so we should remove it from the travis build

### PR DESCRIPTION
**Pull request by pwoods25443**
travis is now passing

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -1,18 +1,25 @@
...
 cache:
   directories:
   - ~/.cache/pip
 
 python:
-- 2.7
 - 3.7
 
 install:
 - pip install -e .
```
Comment (smpiano) [inline .travis.yml:16]: The line `pip install -e .` runs inside the command `docker-compose build`, precisely in the step 5.
So it is assured the invocation of it.
Reference: (https://github.com/GlobalFishingWatch/pipe-tools/blob/66-travis-drop-py27/Dockerfile#L7)
@pwoods25443 Could it be possible to remove this line?

File: .travis.yml [hunk 2]
```diff
@@ -1,18 +1,25 @@
...
+- docker-compose build
 
 script:
 - py.test tests
```
Comment (smpiano) [inline .travis.yml:20]: The line `py.test tests` already runs inside the following command `docker-compose run test`.
This is defined in the `docker-compose.yaml` file, the services section defines a service called `test` and the command is defined here (https://github.com/GlobalFishingWatch/pipe-tools/blob/66-travis-drop-py27/docker-compose.yml#L6).
@pwoods25443 Could it be possible to remove this line?

File: README.md [hunk 3]
```diff
@@ -14,5 +14,13 @@ You can find the registry of changes in the `CHANGES.md` file.
...
-To Run unit tests use the following
-  `docker-compose run tests`
+## Developing
+
+```console
+virtualenv venv
+source venv/bin/activate
+pip install -e .
+py.test tests
+docker-compose run test
```
Comment (smpiano) [inline README.md:24]:
```
py.test tests
￼docker-compose run test
```
There is not mandatory to run both lines, they have the same target that is run `py.test tests`. The first uses the `venv` configured and the second first run the docker-compose build and executed a service declared in `docker-compose.yaml`.

**Review by smpiano — CHANGES_REQUESTED** [review]
@pwoods25443 first, thanks to fix the travis test!.
Could it be possible to use `develop` branch instead of `master` branch as base of the PR?

File: .travis.yml (hunk 1 above)
Comment (pwoods25443) [reply .travis.yml:16]: ok - I'll re-base to `develop` and try removing the `pip install -e` from the travis build

**Comment by pwoods25443** [comment]
@smpiano re-based and ready to merge to `develop`

**Pull request merged**